Python/perfect_number.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct 11 12:38:26 2019
+
+@author: David del Río Medina
+"""
```

Comment:
Not sure if we want to follow this practice. I am not against of it but I would prefer to have a common style maintained for one repository.

Reply:
Done!